### ISSUE TITLE
[Documentation] Spacing documentation

### ISSUE THREAD

**Issue by w3soto** [vuetifyjs/vuetify#8248]
### Environment

**Browsers:** Chrome 73.0.3683.75
**OS:** Linux x86_64

### Steps to reproduce
missing values $spacer * N :

1 - sets the spacing to $spacer
2 - sets the spacing to $spacer
3 - sets the spacing to $spacer
4 - sets the spacing to $spacer
5 - sets the spacing to $spacer

### Expected Behavior
1 - sets the spacing to $spacer * 0.25
2 - sets the spacing to $spacer * .5
3 - sets the spacing to $spacer * 1
4 - sets the spacing to $spacer * 1.5
5 - sets the spacing to $spacer * 3

### Actual Behavior
there is missing $spacer * N description, also for negative values

### Reproduction Link
<a href="https://vuetifyjs.com/en/styles/spacing#spacing" target="_blank">https://vuetifyjs.com/en/styles/spacing#spacing</a>

**Comment by MajesticPotatoe** [comment]
Just above that reads: `The size controls the increment of the property in 4px intervals`

I'll just shorten this to ` 1-12 - sets the spacing to be negative by $spacer * 4px`

**Comment by w3soto** [comment]
You are right, I am new to Vuetify and I am following older video tutorial. Thank you!
